client/src/app/site/pages/meetings/pages/autopilot/components/autopilot/autopilot.component.ts

```diff
@@ -38,6 +38,10 @@ export class AutopilotComponent extends BaseMeetingComponent implements OnInit {
         }
     }
 
+    public get showPollCollection(): boolean {
+        return this.title !== this.translate.instant(`Agenda`);
```

Comment:
Is there maybe a better way to check that?

Reply:
The slide type should have been used here.

Reply:
done, in another pr